common/types/verified_activation.go

```diff
@@ -0,0 +1,54 @@
+package types
+
+import "github.com/spacemeshos/go-spacemesh/log"
+
+type VerifiedActivationTx struct {
+	*ActivationTx
+
+	baseTickHeight uint64
+	tickCount      uint64
+}
+
+// GetWeight of the atx.
+func (vatx *VerifiedActivationTx) GetWeight() uint64 {
```

Comment:
the result should always be the same as ActivationTxHeader. maybe worth factoring this out to a func that takes unit and tickcount. similarly for TickHeight.